docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -4,12 +4,12 @@
 
 ### On this Page
 
-- [See it in Action](#see-it-in-action)

```

Comment:
I'd like to keep see it in action at the top so readers can see the capabilities of the GraphQL API right away. 

We'll definitely need to keep the bits about the playground at the top because that's the only way to explore the graph and see what can be queried at the moment.